src/views/Users/ViewUser/index.jsx

```diff
@@ -0,0 +1,394 @@
+      <Grid item xs={3}>
+        <AutoCompleteText
+          value={defaultToEmptyString(permission.name)}
+          onValueChange={handlePermissionNameChange(permission, index)}
+          getSuggestions={getSuggestions(allPermissions)}
+          label="Name"
+          required
+          disabled={!permission.metadata.isAdditional}
+          inputProps={{
+            autoFocus: true,
+          }}
+        />
+      </Grid>
+      <Grid item xs={4}>
+        <AutoCompleteText
+          multi
+          disabled={!supportsProductRestriction(permission.name)}
+          selectedItems={permission.options.products}
+          onSelectedItemsChange={handleRestrictionChange(
+            permission,
+            'products'
+          )}
+          onValueChange={handleRestrictionTextChange(permission, 'productText')}
+          value={permission.metadata.productText}
+          getSuggestions={getSuggestions(products)}
+          label="Product Restrictions"
+          inputProps={{
+            autoFocus: true,
+          }}
+        />
+      </Grid>
+      <Grid item xs={4}>
+        <AutoCompleteText
+          multi
+          disabled={!supportsActionRestriction(permission.name)}
+          selectedItems={permission.options.actions}
+          onSelectedItemsChange={handleRestrictionChange(permission, 'actions')}
+          onValueChange={handleRestrictionTextChange(permission, 'actionText')}
+          value={permission.metadata.actionText}
+          getSuggestions={getSuggestions(getSupportedActions(permission.name))}
+          label="Action Restrictions"
+          inputProps={{
+            autoFocus: true,
+          }}
+        />
+      </Grid>
+      <Grid item xs={1} className={classes.gridDelete}>
+        <IconButton onClick={() => handlePermissionDelete(permission, index)}>
+          <DeleteIcon />
+        </IconButton>
+      </Grid>
+    </Grid>
+  );
+
+  return (
+    <Dashboard title="Users">
+      {error && <ErrorPanel error={error} />}
+      {isLoading && <Spinner loading />}
+      {!isLoading && (
+        <Fragment>
+          <form autoComplete="off">
+            <Grid container spacing={2}>
+              <Grid item xs>
+                <TextField
+                  required
+                  disabled={!isNewUser}
+                  onChange={handleUsernameChange}
+                  fullWidth
+                  label="Username"
+                  value={username}
+                />
+              </Grid>
+            </Grid>
+            <br />
+            <br />
+            <br />
+            <Typography variant="h5">Roles</Typography>
+            {roles.map(renderRole)}
+            {additionalRoles.map(renderRole)}
+            <Grid item xs className={classes.addGrid}>
+              <Button
+                onClick={handleRoleAdd}
+                className={classes.fullWidth}
+                variant="outlined">
+                <PlusIcon />
+              </Button>
```

Comment:
Let's make the plus button have similar spacing to the required signoff view as well as have its width be similar to the text fields.

![Screen Shot 2019-07-15 at 11 08 03 AM](https://user-images.githubusercontent.com/3766511/61226703-e7a19900-a6f0-11e9-93eb-bf86980459ab.png)

![Screen Shot 2019-07-15 at 11 08 51 AM](https://user-images.githubusercontent.com/3766511/61226736-f8520f00-a6f0-11e9-84f5-7cfb2057634d.png)

This comment also applies to the plus button under "Permissions".